pkg/dashboard/static/index.html

```diff
@@ -138,9 +138,9 @@ <h2 class="m-0 p-1"><img class="m-2 mx-3 me-2" src="static/helm-gray.svg" alt="I
                         Charts (<span></span>)</h2>
                 </div>
                 <div class="bg-secondary rounded-bottom m-0 row p-2">
-                    <div class="col-4 hdr-name">Name</div>
-                    <div class="col-3">Chart Status</div>
-                    <div class="col-2">Chart</div>
+                    <div class="col-6 hdr-name">Name</div>
```

Comment:
Let's not change the layout, it becomes unreadable in important data columns. 
The right strategy is to limit the height of the description box and make it `overflow: hidden`

Reply:
Ack'd and Fixed.